spec/lib/pwned_password_downloader_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+require 'pwned_password_downloader'
+require 'tempfile'
+
+RSpec.describe PwnedPasswordDownloader do
+  let(:destination) { Dir.mktmpdir('pwned_passwords') }
```

Comment:
I like the block form + `around do ...` form because it cleans up the tempdir after the spec is done

Reply:
Ah, I think I missed that when you shared the snippet earlier. I agree it'd be nice to clean up after itself.

Reply:
> I like the block form + `around do ...` form because it cleans up the tempdir after the spec is done

Updated in f457288f2..